src/client.ts

```diff
@@ -887,24 +1096,66 @@ class OptimizelyReactSDKClient implements ReactSDKClient {
    *  @returns {optimizely.OptimizelyConfig | null} optimizely config
    */
   public getOptimizelyConfig(): optimizely.OptimizelyConfig | null {
+    if (!this._client) {
+      logger.warn('Unable to get Optimizely configuration because Optimizely client failed to initialize.');
```

Comment:
Lets call it `config`. We never use the full term `configuration` throughout our ecosystem